cloudformation/batch-substack.cfn.json

```diff
@@ -270,6 +278,14 @@
                 "Ref": "AWS::NoValue"
               }
             ]
+          },
+          "Tags": {
```

Comment:
Shall we also add the Application (parallelcluster-$ClusterName) and the Name (something like AWS Batch Compute) tags?